containers/kubernetes/api-cli/managing-storage.mdx

```diff
@@ -18,175 +18,69 @@ categories:
-    ```bash
-    kubectl apply -f raw-volume/pod.yaml
-    ```
-3. Exec into the container and use the volume as a classic block device:
-    ```bash
-    kubectl exec -it my-awesome-block-volume-app sh
-    / # ls -al /dev/xvda
-    brw-rw----    1 root     disk        8,  32 Mar 23 12:34 /dev/xvda
-    / # dd if=/dev/zero of=/dev/xvda bs=1024k count=100
-    100+0 records in
-    100+0 records out
-    104857600 bytes (100.0MB) copied, 0.043702 seconds, 2.2GB/s
-    ```
-
-## Importing existing Scaleway volumes
-
-1. If you have an already existing volume, with the ID `11111111-1111-1111-111111111111` in the zone `fr-par-1`, you can import it by creating the following PV:
-    ```yaml
-    apiVersion: v1
-    kind: PersistentVolume
-    metadata:
-      name: test-pv
-    spec:
-      capacity:
-        storage: 5Gi
-      volumeMode: Filesystem
-      accessModes:
-        - ReadWriteOnce
-      storageClassName: scw-bssd
-      csi:
-        driver: csi.scaleway.com
-        volumeHandle: fr-par-1/11111111-1111-1111-111111111111
-      nodeAffinity:
-        required:
-          nodeSelectorTerms:
-          - matchExpressions:
-            - key: topology.csi.scaleway.com/zone
-              operator: In
-              values:
-              - fr-par-1
-    ```
-2. Once the PV is created, create a PVC with the same attributes (here scw-bssd as storage class and a size of 5Gi):
-    ```bash
-    kubectl apply -f importing/pvc.yaml
-    ```
-3. Create a pod that uses this volume:
-    ```bash
-    kubectl apply -f importing/pod.yaml
-    ```
-
-## Encrypting volumes
-
-This plugin supports at rest encryption of the volumes with Cryptsetup/LUKS.
+The output of this command provides a quick status update on the CSI plugin within your Kubernetes cluster. For the latest features and enhancements, consider upgrading to [release 0.3](https://github.com/scaleway/scaleway-csi/tree/release-0.3#block-storage-low-latency), which supports **[Block Storage low latency](/storage/block/quickstart/)** volumes.
+
+To identify your current CSI release version, navigate to the [Cockpit interface](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/), specifically the **Kubernetes Cluster - Overview** dashboard.
 
 <Message type="note">
-  Resizing an encrypted volume [does not work](https://github.com/container-storage-interface/spec/issues/445).
+   * In February 2024, block volumes and snapshots management will transition to the [Scaleway Block Storage API](https://www.scaleway.com/en/developers/api/block/). This means, you can view your block volumes in the [Block Storage](https://console.scaleway.com/block-storage/volumes) section of the Scaleway console.
+   * Anticipate the availability of the **boot on block** feature for low-latency Block Storage volumes in Q2.
 </Message>
 
-### Storage class parameters
+## Upgrading to CSI version 0.3
```

Comment:
Is this migration approach still valid after this [change](https://gitlab.infra.online.net/protobuf/protobuf/-/merge_requests/3711) @Tomy2e ??

Reply:
Modified by adding the URL command and CLI information https://github.com/scaleway/docs-content/pull/2642/files#diff-d449ac6de8937fb0dcc564948eacf846056baa7feb258d8bc42fbae937ae80ceR78